docs/deploy-cephfs.md

```diff
@@ -26,6 +26,19 @@ make image-cephcsi
 
 ## Configuration
 
+**NOTE:** To make CephFS CSI driver version >= 1.1.0 work with Ceph v14.2.2
+cluster (not deployed by rook), you need to add settings in the `mgr` section
```

Comment:
(nit) reword slightly?

"To make CephFS CSI driver version >= 1.1.0 work with Ceph v14.2.2 cluster (not deployed by rook), you need to add the following settings, in the `mgr` section of the ceph.conf used by the Ceph manager daemon, and restart the Ceph manager deamon.

```
[mgr]
client mount uid = 0
client mount gid = 0
```

This is due to an [issue](http://tracker.ceph.com/issues/40927) in Ceph v14.2.2 that should be resolved in v14.2.3."

Reply:
Agreed. Reads better. Thanks! Please merge this if it looks OK to you.

Reply:
Approved.

Unfortunately merging is auto-bot driven based on 2 approvals (although there seems to be a time out based call to merge manually at times as well). So, I would wait for one more approval before we merge this, to stick to repository working procedures.

Tagging @Madhu-1 @poornimag @humblec to provide the second approval to move this forward.